.gitignore

```diff
@@ -24,3 +24,4 @@
 **/**/npm-debug.log*
 **/**/yarn-debug.log*
 **/**/yarn-error.log*
+**/**/.vscode
```

Comment:
This is not the correct syntax. This will match "any file named `.vscode` in any folder in this directory", the opposite of what you want.

The file `.vscode/settings.json` is still here. Please double check before you request for another review

Reply:
Since it has already been committed, this will be a multistep process to actually get the file removed (remove it from git ignore, commit, delete the file, commit, update the gitignore to add it back, commit)

Reply:
I nuked it from GitHub's web interface

Reply:
This syntax should work for that (I don't think it needs a '/' at the end). I think the problem was just that the file had already been committed prior to being added to the ignore file.